src/main/java/ConsoleInterface.java

```diff
@@ -0,0 +1,201 @@
+        System.out.println("Got it. I've added this task:");
+
+        Todo todo = new Todo(arguments);
+        taskManager.addTask(todo);
+        todo.print();
+
+        int numTasks = taskManager.getNumTasks();
+        System.out.println("Now you have " + numTasks + " tasks in the list.");
+    }
+
+    /**
+     * Adds a deadline into task manager.
+     *
+     * @param arguments Raw arguments returned by the function {@link #getConsoleInput()}.
+     */
+    public void executeCommandDeadline(String arguments) {
+        System.out.println("Got it. I've added this task:");
+
+        String[] argumentArray = arguments.split("/by");
+        String description = argumentArray[0].trim();
+        String by = argumentArray[1].trim();
+
+        Deadline deadline = new Deadline(description, by);
+        taskManager.addTask(deadline);
+        deadline.print();
+
+        int numTasks = taskManager.getNumTasks();
+        System.out.println("Now you have " + numTasks + " tasks in the list.");
+    }
+
+    /**
+     * Adds an event into task manager.
+     *
+     * @param arguments Raw arguments returned by the function {@link #getConsoleInput()}.
+     */
+    public void executeCommandEvent(String arguments) {
+        System.out.println("Got it. I've added this task:");
+
+        String[] argumentArray = arguments.split("/at");
+        String description = argumentArray[0].trim();
+        String at = argumentArray[1].trim();
+
+        Event event = new Event(description, at);
+        taskManager.addTask(event);
+        event.print();
+
+        int numTasks = taskManager.getNumTasks();
+        System.out.println("Now you have " + numTasks + " tasks in the list.");
+    }
+
+    /**
+     * Executes main interface which interacts with user
+     */
+    public void executeProgram() {
+        while (true) {
+            System.out.println();
+
+            ConsoleInput input = getConsoleInput();
+
+            if (input.getCommand().equals(COMMAND_BYE)) {
+                break;
+            }
+
+            ConsoleInterface.printLineSeparator();
+
+            switch (input.getCommand()) {
+            case COMMAND_LIST:
+                executeCommandList();
+                break;
+            case COMMAND_MARK:
+                executeCommandMark(input.getArguments());
+                break;
+            case COMMAND_UNMARK:
+                executeCommandUnmark(input.getArguments());
+                break;
+            case COMMAND_TODO:
+                executeCommandTodo(input.getArguments());
+                break;
+            case COMMAND_DEADLINE:
+                executeCommandDeadline(input.getArguments());
+                break;
+            case COMMAND_EVENT:
+                executeCommandEvent(input.getArguments());
+                break;
+            default:
+                break;
+            }
+
+            ConsoleInterface.printLineSeparator();
+        }
+    }
+}
```

Comment:
Very organised "main" java file to direct to other java files to task, and to print the respective commands! 😁